src/core_ocean/Registry.xml

```diff
@@ -1629,6 +1631,7 @@
 			<var name="layerThickness"/>
 			<var name="ssh"/>
 			<var name="maxLevelEdgeTop"/>
+			<var name="minLevelEdgeTop"/>
```

Comment:
I think `minLevelEdgeBot` would be the corresponding variable to `maxLevelEdgeTop`.